docs/guides/fan.md

```diff
@@ -26,6 +26,14 @@ In some Macs, the fan has been found to work out of the box. In such a case the
 
         2. Install the package by running `sudo dnf install t2fanrd`
 
+    -   If you’re using Nixos with Flakes;
+
+        1. Add [this](https://github.com/GnomedDev/T2FanRD) repo to your flakes inputs.
```

Comment:
Instead of *this* enter the url directly.

Add <https://github.com/GnomedDev/T2FanRD> to your flakes inputs.